efm32hg-blinky/Makefile

```diff
@@ -132,7 +132,7 @@ deps:
 # REAL TARGETS // COMMON
 #
 GECKO_A_SRC = Gecko_SDK/Device/SiliconLabs/EFM32HG/Source/GCC/startup_efm32hg.S
-GECKO_C_SRC = $(shell find Gecko_SDK/emlib/src -name \*.c) \
+GECKO_C_SRC = $(shell [ -d Gecko_SDK ] && find Gecko_SDK/emlib/src -name \*.c) \
```

Comment:
Is this needed to allow the `make deps` to work when the directory doesn't exist?